Pipfile

```diff
@@ -12,6 +12,10 @@ bandit = "*"
 flake8 = "*"
 
 [packages]
+configargparse = "*"
+gitpython = "*"
+dependency-injector = "*"
+semver = "*"
```

Comment:
These should at least pin major versions IMO

Reply:
Actual versions are pinned in `pipenv.lock`. So far this is highly dev-time version, so I don't think we have to pin versions yet.